roles/openshift_node/defaults/main.yml

```diff
@@ -31,12 +31,9 @@ openshift_node_ami_prep_packages:
 - python-dbus
 - PyYAML
 - yum-utils
-- python2-boto
-- python2-boto3
 - cloud-utils-growpart
 # gluster
 - glusterfs-fuse
-- heketi-client
```

Comment:
why is this removed?

Reply:
This package doesn't exist in origin (centos) repos.

Either this needs to be removed, or we need to refactor how this package is installed, eg, openshift-enterprise vs origin.

I think having this list of packages in defaults/main.yml of openshift_node is a big work-around.  We need to refactor several roles to ensure all the right configs and packages are in place for AMI building, not just bulk-install packages here.

Reply:
ok